packages/vaadin-grid/src/vaadin-grid-column.js

```diff
@@ -204,6 +204,7 @@ export const ColumnBaseMixin = (superClass) =>
      * @protected
      */
     _findHostGrid() {
+      // eslint-disable-next-line @typescript-eslint/no-this-alias
```

Comment:
Decided not to disable this rule for the entire project and disabled it here instead.